helm/values.yaml

```diff
@@ -0,0 +1,34 @@
+psql:
+  jenkinsBuildId: "1"
```

Comment:
what is purpose of this parameter?

Reply:
We use the variable to provide unique name for job that runs backup and it's elb storage in rancher.
This allow us to start parralel db backup process.
1 is deafult, we reassign it from pipeline and it's equal to job build id.